src/pre_defined_classes/mathematical_program/hard/q1/MainTest.java

```diff
@@ -0,0 +1,35 @@
+package pre_defined_classes.mathematical_program.hard.q1;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    // Parsons with distractors
+
+    final double xtExpected = 13100.00;
```

Comment:
Why have this as a variable when it's just used to check the math?

Reply:
I don't really think this is a problem